go.mod

```diff
@@ -5,32 +5,32 @@ go 1.20
 require (
 	github.com/aws/aws-sdk-go v1.44.175
 	github.com/go-logr/logr v1.2.4
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/goombaio/namegenerator v0.0.0-20181006234301-989e774b106e
 	github.com/jetstack/cert-manager v1.7.1
-	github.com/kuadrant/authorino v0.10.0
-	github.com/kuadrant/kuadrant-operator v0.1.1-0.20230323151616-58593d01833a
+	github.com/kuadrant/authorino v0.15.0
```

Comment:
No idea why authorino is in here, separate issue but this controller should not depend directly on it. Seemec to get added here for some reason https://github.com/Kuadrant/multicluster-gateway-controller/commit/a75da134f7595a6600dbe3c48f97bb172a295988#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R11

Reply:
Will be removed here https://github.com/Kuadrant/multicluster-gateway-controller/pull/629/commits/62b95d76a9f19a11c420b81926cfe3e09ff8ace4

Reply:
that one is added automatically by go. when I used the approach above authorino was removed.